suite/jest.config.json

```diff
@@ -1,24 +1,19 @@
 {
     "clearMocks": true,
     "collectCoverage": false,
+    "testEnvironment": "node",
+    "testTimeout": 600000,
+    "verbose": true,
     "extensionsToTreatAsEsm": [".ts"],
-    "globals": {
-        "ts-jest": {
-            "useESM": true
-        }
-    },
-    "moduleFileExtensions": ["ts", "js", "json"],
```

Comment:
This was the line that broke the tests, by giving priority to `.ts` files it caused the multiformat dep to break.

The rest of the updates to this config just bring us inline with latest supported config for jest with ESM modules. https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/